botcity/maestro/impl/v2.py

```diff
@@ -126,22 +126,30 @@ def message(self, email: List[str], users: List[str], subject: str, body: str,
             return model.ServerMessage.from_json(payload=payload)
 
     def create_task(self, activity_label: str, parameters: Dict[str, object],
-                    test: bool = False) -> model.AutomationTask:
+                    test: bool = False, priority: int = 0, min_execution_date=None) -> \
+            model.AutomationTask:
         """
         Creates a task to be executed on the BotMaestro portal.
 
         Args:
             activity_label: The activity unique identified.
             parameters: Dictionary with parameters and values for this task.
             test: Whether or not the task is a test.
+            priority: An integer from 0 to 10 to refer to execution priority.
+            min_execution_date: Minimum execution date for the task.
 
         Returns:
             Automation Task. See [AutomationTask][botcity.maestro.model.AutomationTask]
         """
         url = f'{self._sdk._server}/api/v2/task'
+
         data = {
-            "activityLabel": activity_label, "test": test, "parameters": parameters
+            "activityLabel": activity_label, "test": test, "parameters": parameters, "priority": priority,
+            "minExecutionDate": min_execution_date
         }
+        if min_execution_date and isinstance(min_execution_date, datetime.datetime):
```

Comment:
Se need to verify here if present and not a datetime we need to raise a ValueError